ibis/omniscidb/client.py

```diff
@@ -1183,8 +1184,15 @@ def database(self, name=None):
             )
             return self.database_class(name, new_client)
 
-    def load_data(self, table_name, obj, database=None, **kwargs):
-        """Load data into a given table.
+    def load_data(
+        self,
+        table_name: str,
+        obj: Union[pd.DataFrame, pyarrow.Table],
+        database: Optional[str] = None,
+        method: str = 'rows',
+    ):
```

Comment:
Is removing `**kwargs` intentional?

Reply:
yes, the initial idea for kwarg here was to allow a similar signature for all `load_data` functions across all the backends, and as it seems it is not necessary and it is not used inside the function, I am just removing that for now.